stylelint.config.js

```diff
@@ -7,9 +7,17 @@ module.exports = {
         'declaration-no-important': null, // fixme
         'property-no-vendor-prefix': null, // fixme
         'no-duplicate-selectors': null, // fixme
-        'max-empty-lines': null, // fixme
         'selector-no-vendor-prefix': null, // fixme
         'property-no-unknown': null, // fixme
         'at-rule-no-vendor-prefix': null, // fixme
+        'selector-class-pattern': '[A-Za-z]+([-_]{1,2}[A-Za-z]+)*(_[A-Za-z]+)*$',
```

Comment:
can you explain this?

Reply:
its to match our current className naming convention and make sure any other does not match, but default everything should be kebab case